### PR TITLE
Adds application details view

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This addresses https://github.com/ansible/awx/issues/7236.  Adds support for Application Details view including delete functionality.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION
Screen Shot 2020-06-16 at 2 59 12 PM

### CODE REVIEW

File: awx/ui_next/src/screens/Application/Application/Application.jsx [hunk 1]
```diff
@@ -1,26 +1,139 @@
...
+  const tabsArray = [
+    { name: i18n._(t`Details`), link: `/applications/${id}/details`, id: 0 },
+    { name: i18n._(t`Tokens`), link: `/applications/${id}/tokens`, id: 1 },
+  ];
+
+  let cardHeader = (
+    <TabbedCardHeader>
+      <RoutedTabs tabsArray={tabsArray} />
+      <CardActions>
+        <CardCloseButton linkTo="/teams" />
```
Comment (nixocio) [inline awx/ui_next/src/screens/Application/Application/Application.jsx]:
```suggestion
        <CardCloseButton linkTo="/applications" />
```

File: awx/ui_next/src/screens/Application/Application/Application.test.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+};
+const application = {
+  id: 1,
+  name: 'Foo',
+  summary_fields: {
+    organization: { name: 'Org 1', id: 10 },
+    user_capabilities: { edit: true, delete: true },
+  },
+  url: '',
+  organiation: 10,
```
Comment (nixocio) [inline awx/ui_next/src/screens/Application/Application/Application.test.jsx]:
```suggestion
  organization: 10,
```

**Review by nixocio — CHANGES_REQUESTED** [review]
I got this code locally. Feature seems to be working fine. Good job @AlexSCorey . Minor changes.

**Review by nixocio — APPROVED** [review]

**Review by dsesami — APPROVED** [review]
Bit annoyed with the PF guidelines on capitalization, but approved nonetheless. Looks great!

**Comment by dsesami** [comment]
regate

**Comment by dsesami** [comment]
@AlexSCorey there is one linting issue to be fixed:

```

2020-06-23 17:55:52.195727 | static |
2020-06-23 17:55:52.196036 | static | /awx_devel/awx/ui_next/src/screens/Application/Application/Application.jsx
2020-06-23 17:55:52.196129 | static |   21:10  error  TabbedCardHeader not found in '../../../components/Card'  import/named
2020-06-23 17:55:52.196145 | static |
2020-06-23 17:55:52.196185 | static | ✖ 1 problem (1 error, 0 warnings)
```

**Pull request merged**